src/fragments/start/getting-started/flutter/add-api.mdx

```diff
@@ -22,9 +22,9 @@ On the creation form:
 
     ![Create app backend](/images/lib/getting-started/flutter/connect-to-cloud-create-app-backend.png)
 
-4. The on screen text should walk you through the deployment progress and, when the deployment status reads **Deployment completed**, click **Launch Studio**.
+4. The on-screen text should walk you through the deployment process and, when the deployment status reads **Deployment completed**, click **Launch Studio**.
```

Comment:
This still reads a bit off to me: 
- usage of `should` implies that there is potential that the on-screen text will not walk the reader through the deployment process
- the comma after `Deployment created` is extraneous
- imo this is two sentences, the first being informational, and the second action oriented

```suggestion
4. The on-screen text will walk you through the deployment process. Once the deployment status reads **Deployment completed** click **Launch Studio**.
```